.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
JSON does not officially support comments. Does this work?

Reply:
comment is not described in json spec, but node js and dotnet json parsers treat // as comment.